src/core/core.scale.js

```diff
@@ -857,11 +857,17 @@ module.exports = function(Chart) {
 
 					var label = itemToDraw.label;
 					if (helpers.isArray(label)) {
-						for (var i = 0, y = 0; i < label.length; ++i) {
+						var lineCount = label.length / 2;
+						var lineHeight = tickFont.size * 1.5;
+						var y = 0;
```

Comment:
More concise notation:
```
var y = me.isHorizontal() ? 0 : -lineHeight * lineCount;
```

But also, I think you need `lineHeight` instead of `tickFont.size`

![image](https://user-images.githubusercontent.com/3874900/36020483-2c0ba378-0d83-11e8-815f-336e9f9d0197.png)

Reply:
When I do: 
`var lineCount = label.length / 2;`
`var lineHeight = tickFont.size * 1.5;`
`var y = me.isHorizontal() ? 0 : -lineHeight * lineCount;`

I get: 
![lineheight](https://user-images.githubusercontent.com/27080760/36031716-b42bb96e-0d79-11e8-86c4-62af85123fac.JPG)

[JSFiddle](https://jsfiddle.net/qnmy4soo/1/)

Which seems just slightly off. I think the equation is missing something small or I'm doing it wrong.